### DISCUSSION TITLE
Multiple Updates in a single Query

### DISCUSSION THREAD

**Discussion by Mrjuanblack** [linq2db/linq2db#3755] [Q&A] [answered]
Hello
I've been checking out the documentation pages about this specific issue. I saw that there's the `MultiInsert` that is really handy and easy to use, and I need something similar for updating multiple tables in a single query.

There's something called `Merge` and it could possibly help me, but I'm using Postgres 14 and I think `Merge` support is only for Postgres 15 right now.
Is there any other way to update multiple tables in a single query? I really don't want to update each table in different queries.

**Comment by MaceWindu** [comment, marked as answer]
What kind of SQL you expect? `MultiInsert` is Oracle-specific feature, other databases don't have it.

**Reply by Mrjuanblack** [reply]
Yep, I took some time to read about the topic, the standard is to modify each table in a transaction. Thanks for the answer though :)
